### ISSUE TITLE
Problem with moving/copying files 

### ISSUE THREAD

**Issue by jony0008** [linuxmint/nemo#1450]
When selecting files and Ctrl+C to copy, then opening a folder and clicking Ctrl+V to paste files nothing happens. Why are the files not copied? I am using Cinnamon Desktop on Debian Stretch.

**Comment by shreyash14s** [comment]
Check whether you have write permission on the directory where you are pasting the files.

**Comment by jony0008** [comment]
I have, it always works the second time a do it. I always have to do it at least twice.

**Comment by jony0008** [comment]
The same thing happens when I open a new tab

**Comment by kj800x** [comment]
@jony0008 Is this problem still occurring? I used to have an issue with copying and I noticed that it was when I would do a CTRL+a, CTRL+c to copy sometimes it wouldn't actually copy. I investigated a little more and discovered that sometimes I was doing "Hold CTRL, Press a, Press c, Release CTRL" (which wouldn't work), and sometimes I was doing "Hold CTRL, Press a, Release CTRL, Hold CTRL, Press c, Release CTRL" (which would work).

I updated yesterday and noticed that this is fixed now.
